skbio/stats/tests/test_composition.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import absolute_import, division, print_function
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from unittest import TestCase, main
+
+import numpy as np
+from numpy.testing import assert_array_almost_equal
+from skbio.stats.composition import (_closure, multiplicative_replacement,
+                                     perturb, perturb_inv, power,
+                                     clr, centralize)
+
+
+class CompositionTests(TestCase):
+
+    def setUp(self):
+        self.data1 = np.vstack((
+            np.array([2, 2, 6]),
+            np.array([4, 4, 2])))
+        self.data2 = np.array([2, 2, 6])
```

Comment:
The array can be directly created in a cleaner way as 
```
np.array([[2, 2, 6],
          [4, 4, 2]])
```
instead of
```
np.vstack((np.array([2, 2, 6]),
           np.array([4, 4, 2])))
```
(there are several instances of that)